src/ui/src/js/run.js

```diff
@@ -275,13 +298,23 @@ export default function appRun(
     }
   });
 
-  $interval(function() {
-    EventService.connect(TokenService.getToken());
-  }, 5000);
-
-  $interval(function() {
-    TokenService.preemptiveRefresh();
-  }, 30000);
+  $interval(
```

Comment:
These intervals were triggering the angular `digest()`, which re-checks and potentially re-renders everything, which meant the permission checks in the ng-show that I added all over the place would happen every interval for no real compelling reason.  Since neither interval should result in data changes that would make `digest()` necessary to call, I just updated these to not do that (the `false` that gets passed in now).